<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fp</class>
 <widget class="QDialog" name="fp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1856</width>
    <height>1278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fp</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="start_fp">
      <property name="text">
       <string>start_fp</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>82</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="set_button">
      <property name="text">
       <string>set</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>183</width>
     <height>143</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="slotLabel">
      <property name="text">
       <string>slot</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="cLBLabel">
      <property name="text">
       <string>CLB</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="bRAMLabel">
      <property name="text">
       <string>BRAM</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="bRAMLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="dSPLabel">
      <property name="text">
       <string>DSP</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="dSPLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="cLBLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QComboBox" name="comboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="enter_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="num_slotsLabel_2">
      <property name="text">
       <string>num_slots</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="num_slotsLineEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>1161</width>
     <height>1101</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>1495</x>
     <y>40</y>
     <width>351</width>
     <height>611</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FPGA">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>FPGA</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="1" column="0">
     <widget class="QLabel" name="utilLabel">
      <property name="text">
       <string>Util</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="utilLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="set_util">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
